test/foundry/OrderBook/unit/MarketOrder.t.sol

```diff
@@ -414,4 +414,378 @@ contract OrderBookMarketOrderUnitTest is Test, CloberMarketSwapCallbackReceiver
         assertEq(inputAmount, 0, "ERROR_AMOUNT_IN");
         assertEq(outputAmount, 0, "ERROR_AMOUNT_OUT");
     }
+
+    function testTradeLogWithBidsInSameBlocks() public {
+        _createOrderBook(0, 0);
+
+        _createPostOnlyOrder(Constants.ASK);
+
+        orderBook.marketOrder({
+            user: Constants.TAKER,
+            limitPriceIndex: Constants.LIMIT_BID_PRICE,
+            rawAmount: Constants.RAW_AMOUNT,
+            baseAmount: 0,
+            options: _buildMarketOrderOptions(Constants.BID, Constants.EXPEND_INPUT),
+            data: new bytes(0)
+        });
+
+        CloberOrderBook.BlockTradeLog memory tradeLog = orderBook.blockTradeLogs(0);
+        assertEq(tradeLog.blockTime, 1, "ERROR_BLOCK_TIME");
+        assertEq(tradeLog.askVolume, 0, "ERROR_ASK_VOLUME");
+        assertEq(tradeLog.bidVolume, Constants.RAW_AMOUNT, "ERROR_BID_VOLUME");
+        assertEq(tradeLog.open, Constants.PRICE_INDEX, "ERROR_OPEN");
+        assertEq(tradeLog.close, Constants.PRICE_INDEX, "ERROR_CLOSE");
+        assertEq(tradeLog.high, Constants.PRICE_INDEX, "ERROR_HIGH");
+        assertEq(tradeLog.low, Constants.PRICE_INDEX, "ERROR_LOW");
+
+        // take will be in same block
+        _createPostOnlyOrder(Constants.ASK);
```

Comment:
I believe changing to a different `priceIndex` here would be better.

Reply:
@JhChoy This case has already been coveraged at `testTradeLogWith*ForPrice*crease`